legion/engine/core/time/time_span.hpp

```diff
@@ -125,13 +125,13 @@ namespace legion::core::time
             return time_span(0);
         }
 
-        L_NODISCARD static constexpr time_span(min)() noexcept
+        L_NODISCARD static constexpr time_span min() noexcept
         {
             // get minimum value
             return time_span((std::chrono::duration_values<time_type>::min)());
         }
 
-        L_NODISCARD static constexpr time_span(max)() noexcept
+        L_NODISCARD static constexpr time_span max() noexcept
         {
```

Comment:
did this compile before ? because it is a potential fix for issues like these:
https://stackoverflow.com/a/2789509/5487005

Reply:
not sure i think it was a merge thing